packages/fault-detector/src/service.ts

```diff
@@ -41,7 +42,7 @@ export class FaultDetector extends BaseServiceV2<Options, Metrics, State> {
         },
         startBatchIndex: {
           validator: validators.num,
-          default: 0,
+          default: -1,
```

Comment:
`BaseServiceV2` doesn't cleanly handle "undefined" defaults but -1 is good enough here.